OsmSharpMacMSTest/OsmSharpMacMSTest.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
It seems like you've added a test project but we already have a test project. Why is it better to have two?